rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

```diff
@@ -27,9 +27,20 @@ Machine pools are a higher level construct to compute machine sets.
 
 A machine pool creates compute machine sets that are all clones of the same configuration across availability zones. Machine pools perform all of the host node provisioning management actions on a worker node. If you need more machines or must scale them down, change the number of replicas in the machine pool to meet your compute needs. You can manually configure scaling or set autoscaling.
 
-By default, a cluster is created with one machine pool. You can add additional machine pools to an existing cluster, modify the default machine pool, and delete machine pools.
+By default, a cluster is installed with one machine pool. You can label that machine pool during installation. The `Default` machine pool node type or size cannot be changed after the cluster is installed, and that machine pool cannot be deleted. 
 
-Multiple machine pools can exist on a single cluster, and they can each have different types or different size nodes.
+After a cluster's installation:
+
+* You can relabel a `Default` machine pool.
+* You can add additional machine pools to an existing cluster. Also, you can label these machine pools. 
++
+[NOTE]
+====
+You cannot change the machine pool's node type or size. The machine pool node type or size is specified during their creation only. If you need a machine pool with a different node type or size, re-create a machine pool with a new node type or size.
```

Comment:
"The" to "a" as more than one machine pool can exist. 

Also, convert "machine pool" as a possessive (machine pool's") in the second sentence. 

I think the last two sentences could be tightened up with something like the following suggestion:

> During cluster installation, the installation program specificies a machine pool's node type and size. After cluster installation, you cannot change the node type or size for the machine pool. If you need a different node type and size, you must recreate a machine pool and specify the required node type and size values.

Reply:
For first sentence, I am changing **machine pool's node** -> **machine pool node** to maintain the consistency.